README.md

```diff
@@ -71,11 +71,11 @@ There are many plugins available within the Nautobot Apps ecosystem. The below s
 
 The best platform for general feedback, assistance, and other discussion is our [GitHub discussions](https://github.com/nautobot/nautobot/discussions). To report a bug or request a specific feature, please open a GitHub issue using the [appropriate template](https://github.com/nautobot/nautobot/issues/new/choose).
 
-If you are interested in contributing to the development of Nautobot, please read our [contributing guide](CONTRIBUTING.md) prior to beginning any work.
+If you are interested in contributing to the development of Nautobot, please read our [contributing guide](https://docs.nautobot.com/projects/core/en/stable/development/) prior to beginning any work.
```

Comment:
This should probably point to https://docs.nautobot.com/projects/core/en/stable/development/core/#contributing

Reply:
I feel like this should take the user to https://docs.nautobot.com/projects/core/en/stable/development/. Because they can be contributing via apps/core/jobs. Is this link restricted to core only?

Reply:
The apps should have their own contributing guidelines that they post in their repositories. Since this lives in the Nautobot repo it should take you to the docs for contributing to Nautobot core.